api/v1alpha1/releaseplan_types.go

```diff
@@ -51,6 +51,11 @@ type ReleasePlanSpec struct {
 	// +optional
 	ServiceAccount string `json:"serviceAccount,omitempty"`
 
+	// ReleaseGracePeriodDays is the number of days a Release should be kept
+	// This value is used to define the Release ExpirationTime
```

Comment:
https://book.kubebuilder.io/reference/markers/crd-validation is what I meant with my comment on the constants file. Maybe we can just add the kubebuilder annotation here and not need the webhook change (or constants file)

Reply:
Interesting. 
```
 +kubebuilder:default any
sets the default value for this field.
```

I will try it.

Reply:
The good thing about having the constant is that it can be re-used eventually. But not 100% sure that we would re-use it (besides tests).

Reply:
Changed to set the default through `kubebuilder`